File: src/main/java/io/asadh/intellij_twirl/language/TwirlLanguage.java
package io.asadh.intellij_twirl.language;

import com.intellij.lang.Language;

public class TwirlLanguage extends Language {
  public static final TwirlLanguage INSTANCE = new TwirlLanguage();

  private TwirlLanguage() {
    super("Twirl");
  }
}

File: src/main/kotlin/io/asadh/intellij_twirl/language/TwirlLanguage.kt
package io.asadh.intellij_twirl.language

import com.intellij.lang.Language

class TwirlLanguage : Language("Twirl") {
  companion object {
    @JvmField
    val INSTANCE = TwirlLanguage()
  }

  fun getName() = "Twirl"
}
